statistics/handle/handle.go

```diff
@@ -1944,3 +1944,30 @@ func (h *Handle) GetPredicateColumns(tableID int64) ([]int64, error) {
 	}
 	return columnIDs, nil
 }
+
+// RecordHistoricalStatsToStorage records the given table's stats data to mysql.stats_history
+func (h *Handle) RecordHistoricalStatsToStorage(dbName string, tableInfo *model.TableInfo) (uint64, error) {
+	ctx := context.Background()
+	blockSize, _ := mysql.GetDefaultFieldLengthAndDecimal(mysql.TypeLongBlob)
```

Comment:
That's not correct, the returned value is `4294967295` but our limit is `6MB` (https://docs.pingcap.com/zh/tidb/dev/tidb-limitations/#%E5%8D%95%E5%88%97%E7%9A%84%E9%99%90%E5%88%B6).
We should define a new constant for this.